worlds/tloz/Options.py

```diff
@@ -33,8 +33,17 @@ class StartingPosition(Choice):
     option_dangerous = 2
     option_very_dangerous = 3
 
+class ArrowLocation(Choice):
+    """Bow requires you to also find an Arrow in order to use it.
+    Shop means that Arrow is guaranteed to appear in one of the shops in your world.
+    Anywhere shuffles Arrow into the pool to be placed in any game in the multiworld."""
+    display_name = "Arrow Location"
+    option_shop = 0
+    option_anywhere = 1
```

Comment:
```suggestion
class ArrowLocation(Choice):
    """Choose whether the Arrow will restricted to the shop or placed anywhere in the multiworld.
    Bow requires you to also find an Arrow in order to use it."""
    display_name = "Arrow Location"
    option_shop = 0
    option_anywhere = 1
```
Something like this maybe?

Also, since this is an option with two choices, maybe it could be reworded into a Toggle instead?
Something like:
```
class ArrowLocation(Toggle):
    """Choose whether the Arrow will restricted to the shop or placed anywhere in the multiworld.
    Bow requires you to also find an Arrow in order to use it."""
    display_name = "Arrow in Shop"
```
And then flipping it in the code (since now option 0 is anywhere and option 1 is in the shop)

Reply:
It was originally a choice because I had in mind a potential expansion to have "start with" also be an option for arrow. That seemed like a reasonable expansion of the feature that might be reasonably requested.